easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb

```diff
@@ -1761,11 +1763,6 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/M/ML/MLEHMANN/'],
         'checksums': ['a86a1c4ca4f3006d7479064425a09fa5b6689e57261fcb994fe67d061cba0e7e'],
     }),
-    ('IO::Compress::Bzip2', '2.096', {
```

Comment:
@akesandgren Can you clarify why this has to be removed?

Reply:
@akesandgren ping?

Reply:
It's part of Perl main code if I remember correctly now....

Reply:
And if left something failed in the tests due to this, probably something about incompatible bzip/zip/... levels